ffs/coreipfs/coreipfs.go

```diff
@@ -1,13 +1,12 @@
 package coreipfs
```

Comment:
Changed opinion here. Decided to pre-fill pinset cache on `New`. This saves work while calling hot APIs.